soknad/src/main/java/no/nav/k9/søknad/felles/aktivitet/Arbeidstaker.java

```diff
@@ -81,68 +62,7 @@ public Builder norskIdentitetsnummer(NorskIdentitetsnummer norskIdentitetsnummer
         public Arbeidstaker build() {
             return new Arbeidstaker(
                 norskIdentitetsnummer,
-                organisasjonsnummer,
-                perioder);
-        }
-    }
-
-    @JsonIgnoreProperties(ignoreUnknown = true)
-    @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.NONE, getterVisibility = JsonAutoDetect.Visibility.NONE, setterVisibility = JsonAutoDetect.Visibility.NONE, isGetterVisibility = JsonAutoDetect.Visibility.NONE, creatorVisibility = JsonAutoDetect.Visibility.NONE)
-    public static final class ArbeidstakerPeriodeInfo {
-        /**
-         * En prosent mellom 0 og 100 som sier hvor mye søkeren kan jobbe (for en arbeidsgiver i en periode)
-         * i forhold til hva den normalt ville gjort (om den ikke hadde hatt behov for pleiepenger).
-         * - Ved avkorting mot arbeid er det tap av inntekt som skal dekkes. Hvilken stillingsprosent
-         * eller hvor mange timer man jobber er ikke relevant, men er faktorer som kan brukes til å utlede "skalJobbeProsent".
-         * - Om man har en stillingsprosent på 40%, men i perioden kan dekke en 20% stilling skal "skalJobbeProsent" være 50%.
-         * - Om man normalt jobber 30 timer i uken, men nå får jobbet 10 timer skal "skalJobbeProsent" være 33.33%.
-         * - Gitt at pleiepengene blir avkortet mot arbeid (ikke gradert mot tilsyn) vil "skalJobbeProsent" brukes ved
-         * håndtering av inntektsmelding fra den aktuelle arbeidsgiveren for å regne ut inntektstapet i perioden.
-         */
-        @JsonProperty(value = "skalJobbeProsent")
-        @DecimalMin(value = "0.00", message = "skalJobbeProsent [${validatedValue}] må være >= {value}")
-        @DecimalMax(value = "100.00", message = "skalJobbeProsent [${validatedValue}] må være <= {value}")
-        public final BigDecimal skalJobbeProsent;
```

Comment:
@ramrock93  hva samler dere inn i brukerdialogen?  
ser på muligheter for å standardisere oppgitt arbeidstid på tvers av omsorgspenger/ pleiepenger.

Reply:
Er ikke helt landet enda. Forslaget nå er at vi får vite hva som er "normal arbeidstid" for forholdet på timer per dag for hele søknadsperioden. Hvis det er varierende blir det et gjennomsnitt.  
Og i perioder hvor søker arbeider blir det periodisert med informasjon om faktisk arbeid på timer per dag i perioden som søker setter. Denne perioden må være innenfor søknadsperioden, kan være flere